modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/ReadonlyToolbarResizeTest.ts

```diff
@@ -0,0 +1,118 @@
+import { Pipeline, Chain, Log, NamedChain, ApproxStructure, Assertions, StructAssert, Mouse, UiFinder, Guard } from '@ephox/agar';
+import { UnitTest } from '@ephox/bedrock-client';
+import { TinyLoader } from '@ephox/mcagar';
+import SilverTheme from 'tinymce/themes/silver/Theme';
+import { cResizeToPos } from '../../../module/UiChainUtils';
+import { Element, Body } from '@ephox/sugar';
+import Editor from 'tinymce/core/api/Editor';
+
+UnitTest.asynctest('Editor (Silver) Configuration Close test', (success, failure) => {
```

Comment:
This name looks to be incorrect? as it's not related to what's happening in the test.

- [ ] Make sure test name is correct